Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -968,175 +968,175 @@ projectList=Projektliste
-setRepresentative=Repr\u00E4sentant setzen\:
+setImportConfigurationConfirmMessage=Die Importkonfiguration wird für alle ausgewählten Vorgänge gesetzt. Diese Aktion kann nicht rückgängig gemacht werden. Möchten Sie fortfahren?
+setImportConfigurationForSelectedProcessesText=Bitte wählen Sie die Importkonfiguration für die selektierten Vorgänge aus. Der Re-Import von Katalogmetadaten ist nur für Vorgänge mit Importkonfiguration vom Typ 'Katalogsuche' möglich.
+setRepresentative=Repräsentant setzen\:
 settings=Einstellungen
-sheetCounting=Blattz\u00E4hlung
-sheetCountingRectoVerso=Blattz\u00E4hlung recto-verso
-shortcuts=Tastaturk\u00FCrzel
-shortcuts.change=Tastaturk\u00FCrzel anpassen
+sheetCounting=Blattzählung
+sheetCountingRectoVerso=Blattzählung recto-verso
+shortcuts=Tastaturkürzel
+shortcuts.change=Tastaturkürzel anpassen
 shortcuts.clear=Tastenzuweisung entfernen
 shortcuts.dataEditor.help=Hilfe anzeigen
 shortcuts.dataEditor.structuredView=Strukturierte Ansicht anzeigen
 shortcuts.dataEditor.detailView=Detailansicht anzeigen
 shortcuts.dataEditor.nextItem=Ein Element weiter springen
-shortcuts.dataEditor.previousItem=Ein Element zur\u00FCck springen
+shortcuts.dataEditor.previousItem=Ein Element zurück springen
 shortcuts.dataEditor.nextItemMulti=Mehrere Elemente weiter springen
-shortcuts.dataEditor.previousItemMulti=Mehrere Elemente zur\u00FCck springen
+shortcuts.dataEditor.previousItemMulti=Mehrere Elemente zurück springen
 shortcuts.dataEditor.downItem=Ein Element nach unten
 shortcuts.dataEditor.upItem=Ein Element nach oben
 shortcuts.dataEditor.downItemMulti=Mehrere Elemente nach unten
 shortcuts.dataEditor.upItemMulti=Mehrere Elemente nach oben
 show=Anzeigen
 showAssignedPages=Zeige zugewiesene Seiten
-showClosedProcesses=Abgeschlossene Vorg\u00E4nge anzeigen
+showClosedProcesses=Abgeschlossene Vorgänge anzeigen
 showDataForProcess=Zeige Daten des Vorgangs
 showHierarchyLevel=Zeige Hierarchielevel
 showInSelectList=In Auswahlliste anzeigen
 showInactiveProjects=Deaktivierte Projekte zeigen
 showOnlyStructureElements=Zeige nur Strukturelemente
-showProjectProcesses=Vorg\u00E4nge dieses Projektes anzeigen
+showProjectProcesses=Vorgänge dieses Projektes anzeigen
 showProjectTasks=Aktuelle Aufgaben dieses Projektes anzeigen
 signIn=Anmelden
 # shouldContentBeRemoved is used in onclick
-shouldContentBeRemoved=Soll der Inhalt wirklich gel\u00F6scht werden?
-shouldItemBeRemoved=Soll dieser Eintrag wirklich gel\u00F6scht werden?
+shouldContentBeRemoved=Soll der Inhalt wirklich gelöscht werden?
+shouldItemBeRemoved=Soll dieser Eintrag wirklich gelöscht werden?
 showProcessInTaskList=Spalte der Vorgangsdaten in eigenen Aufgaben anzeigen
 sorting=Sortierung
 surname=Nachname
 save=Speichern
-saveExit=Speichern \u0026 schlie\u00DFen
+saveExit=Speichern & schließen
```

Comment:
I don't know whether the ampersand needs special treatment. Maybe all four similar replacements are wrong.

Reply:
This works without a problem, because the ampersand is not directly included in the XHTML page:

<img width="247" alt="Bildschirmfoto 2025-05-15 um 08 09 08" src="https://github.com/user-attachments/assets/545e9a71-6880-40d3-bd54-f254b118900f" />

Reply:
Thanks for testing, so this is resolved.